spring-native-image/src/main/java/com/example/benchmarks/jibber/JibberController.java

```diff
@@ -3,15 +3,15 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
-import io.micrometer.core.annotation.Timed;
+// import io.micrometer.core.annotation.Timed;
 
 /**
  * REST Controller which serves as an entry-point for requests for jibber nonsense verse.
  *
  */
 @RestController
 @RequestMapping("/jibber")
-@Timed
+// @Timed
```

Comment:
Why is this commented out?

Reply:
I forgot to uncomment. Thanks!

Reply:
Followed in https://github.com/graalvm/graalvm-demos/pull/168/commits/5ad973dc312a856a73b47064964b9550ddfdf442